producer/writer/options.go

```diff
@@ -33,7 +33,8 @@ import (
 
 const (
 	defaultDialTimeout               = 10 * time.Second
-	defaultMessageRetryBackoff       = 5 * time.Second
```

Comment:
Hm is this not used? How did it get past metalint?

Reply:
It was used, just renamed

Reply:
Oh I see, sounds good